docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-library-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='using-standalone-perspectives-library-proc']
+= Using standalone library perspective
+
+You can use the library perspective of {CENTRAL} to select a project you want to edit. You can also perform all the authoring functions on the selected project. The home page of the standalone library perspective is the *Projects* tab of the *MySpace* page.
+
+Further the standalone library perspective can be used in two ways, _with_ and _without_ using the `header=UberfireBreadcrumbsContainer` parameter. The difference between the two ways is that the link with the *_header_* parameter will display a breadcrumbs link on top of the library perspective. Using this link you can create additional *Spaces* for your projects.
```

Comment:
I would remove "Further" from the 1st sentence of the 2nd paragraph. Just say "The standalone library.."

Also, I would change: "The difference between the two ways is that the link with the.." to 
The difference is that the link with the..

Reply:
Changed as suggested.